auth/provide.go

```diff
@@ -42,6 +42,17 @@ func ProvidePrimaryServerChain(apiBase string) fx.Option {
 		))
 }
 
+// accessLevelAttributeKeyAnnotated shares the access level attribute key which outside packages such as argus/store can
+// as parameters.
+func accessLevelAttributeKeyAnnotated() fx.Annotated {
+	return fx.Annotated{
+		Name: "access_level_attribute_key",
```

Comment:
💯  Both options seemed pretty odd for me at the time I added it. Does uber/fx have a standard? It seemed fine to me having `argus/auth` emitting the value since it originates there 🤷‍♂️

Reply:
yeah I'm good with that 👍 this is like one of the artifacts it's providing.